packages/terra-menu/src/_MenuUtils.js

```diff
@@ -0,0 +1,57 @@
+const getPopupHeight = (contentHeight) => {
+  if (contentHeight <= 40) {
```

Comment:
I've looked at this some. And maybe I'm missing something obvious, but I'm not seeing how those could be used here to get rid of the if statement.